terraform/test_cluster/variables.tf

```diff
@@ -264,3 +274,28 @@ variable "staging_new_relic_app_name" {
   type        = string
   default     = "stela-api-staging"
 }
+
+variable "dev_cloudfront_url" {
```

Comment:
I am *not* a terraform expert by any means -- do you know if we need to repeat these variable definitions across test / prod cluster, or might there be a pattern for sharing this kind of variable that is universal to all environments / clusters.

Reply:
I've looked a bit, and I haven't found a way to do this for the variables files where the variables are declared. There is a way to do this with the variable _assignments_ in Terraform Cloud, but even that has limited utility for us, because of the way our test environments are in the same workspace (so usually there aren't duplicate variables across workspaces, the test one will have `dev_cloudfront_url` and `staging_cloudfront_url`, while the prod one will just have `cloudfront_url`). The reason the workspaces are organized this way is so we can save money by running staging and dev on the same EKS cluster.